test/Riok.Mapperly.Tests/Mapping/ExtensionMethodTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Riok.Mapperly.Tests.Mapping;
+
+[UsesVerify]
+public class ExtensionMethodTest
+{
+    [Fact]
+    public Task UpdateMethod()
+    {
+        var source = TestSourceBuilder.MapperWithBodyAndTypes(
+            """static partial void MapToB(this A source, B target);""",
```

Comment:
Don't think raw strings are needed here. I'd change this to `"partial void MapToB(this A source, B target);"` and `"class A { public int Value { get; set; } }"`, etc to match the other tests.